packages/api/tests/endpoints/test-executions.js

```diff
@@ -2,78 +2,266 @@
 
 const test = require('ava');
 const request = require('supertest');
-const awsServices = require('@cumulus/aws-client/services');
+const cryptoRandomString = require('crypto-random-string');
```

Comment:
Was this changeset a git mv from test/endpoints/execution.js plus additions, or were there updates to existing tests?

Reply:
Yeah the former. There were no updates to existing tests.